packages/shader-lab/src/contentParser/ShaderContentParser.ts

```
@@ -56,19 +56,11 @@ export class ShaderContentParser {
-    return RenderStateType.includes(token.type);
-  }
-
-  private static _isEngineType(token: BaseToken) {
-    return EngineType.includes(token.type);
-  }
-
-  private static _symbolTable: SymbolTableStack<ISymbol, SymbolTable> = new SymbolTableStack();
+  private static _symbolTableStack: SymbolTableStack<ISymbol, ContentSymbolTable> = new SymbolTableStack();
 
   static reset() {
     this._errors.length = 0;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)